rec: Remove XPF support (#11856)
PowerDNS/pdns: pdns/recursordist/docs/settings.rst

------------------
.. versionadded:: 4.2.0

.. versionchanged:: 4.8.0
   This setting was removed.

-  IP addresses or netmasks, separated by commas
-  Default: empty

.. note::
  This is an experimental implementation of `draft-bellis-dnsop-xpf <https://datatracker.ietf.org/doc/draft-bellis-dnsop-xpf/>`_.

>   This is a deprecated feature that was removed starting with version 4.8.0.

Reviewer: ```suggestion
  This deprecated feature was removed in version 4.8.0.
```
?
